CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ option(TRITON_ENABLE_ONNXRUNTIME_OPENVINO
 set(TRITON_BUILD_CONTAINER "" CACHE STRING "Triton container to use a base for build")
 set(TRITON_BUILD_CONTAINER_VERSION "" CACHE STRING "Triton container version to target")
 set(TRITON_BUILD_ONNXRUNTIME_VERSION "" CACHE STRING "ONNXRuntime version to build")
+set(TRITON_BUILD_ONNXRUNTIME_CONFIG "" CACHE STRING "ONNXRuntime config to build. Debug, Release, RelWithDebInfo")
```

Comment:
Can you just use CMAKE_BUILD_TYPE instead of introducing a new variable.

Reply:
My initial intention to keep this separate was to enable building the bridge and backend in different modes. Rethinking this such an ability is not necessary... Will merge it.